server/views/formPages/timeMacro.njk

```diff
@@ -6,80 +6,61 @@
 
 {% macro timeInput(params) %}
 
-    {#- a record of other elements that we need to associate with the input using
-    aria-describedby – for example hints or error messages -#}
-    {% set describedBy = params.fieldset.describedBy if params.fieldset.describedBy else "" %}
+<div class="govuk-form-group {%- if params.errorMessage %} govuk-form-group--error {% endif %}">
+    {% call govukFieldset({
+            role: 'group',
+            legend: {
+                text: "Time",
+                isPageHeading: false
+            }
+        }) 
+    %}
+
+    {{
+    govukHint({
+        id: 'incidentDate[time]-hint',
```

Comment:
I think we need to change the name of the file/macro - the names suggest a generic time macro but we've now hardcoded it to only refer to incident date time

Reply:
I might have been tempted to keep it generic and pass in an id, and name, value props for hours/minutes..

Reply:
1. file and macro name changed
2. id and name values now passed in to macro rather than being hard coded